contentreview-impl/tool/src/java/org/sakaiproject/contentreview/servlet/GradingCallbackServlet.java

```diff
@@ -136,6 +136,10 @@ protected void doPostXml(HttpServletRequest request, HttpServletResponse respons
 			ContentReviewItem cri = contentReviewService.getFirstItemByContentId(sourcedId);
 			if(cri == null){
 				M_log.debug("Could not find the content review item for content " + sourcedId);
+				cri = contentReviewService.getFirstItemByExternalId(sourcedId);
+			}
+			if(cri == null){
+				M_log.debug("Could not find the content review item for paper id " + sourcedId);
```

Comment:
Wouldn't it make sense to log this as a warning as we're not expecting this under normal operation are we?